test/luatest_helpers/misc.lua

```diff
@@ -22,6 +22,7 @@ local function skip_if_enterprise()
 end
 
 return {
+    is_debug_build = is_debug_build,
```

Comment:
Mmm? Shall we rework all ERRNIJ-based luatests with the same assertion?

Reply:
I wouldn't bother. Might ask to do it in new tests. Besides, it makes sense to do not in tests themselves, which anyway usually call `skip_if_not_debug()`, but rather in luatest helpers which use errinjs. From whose API it is not always clear that they work only in debug.